ondemand/README.md

```diff
@@ -1,4 +1,389 @@
+to navigate to the sandbox app workspace.
+
+IMG NEEDED
+
+Now create a new app from the button labeled "New App".
+
+IMG NEEDED
+
+This will bring you to a page where you'll click "Clone Existing App" which will bring you to
+this form to fill out.  
+
+Fill in `jupyter` as the directory name. `/var/git/bc_example_jupyter` as the Git Remote and
+check "Create a new Git Project from this?".  Then click "Submit" to create a new development
+application.
+
+IMG NEEDED
+
+This copied what was in `/var/git/bc_example_jupyter` to `/home/hpcadmin/ondemand/dev/jupyer`. 
+You can navigate to these files [through the Files app with this link](https://localhost:3443/pun/sys/files/fs/home/hpcadmin/ondemand/dev/jupyter/)
+or simply Press the "Files" button in Jupyter's row of the sandbox applications table.
+
+IMG NEEDED
+
+You'll also need to setup `git` for the hpcadmin user at this point.
+
+Configure your user email and name so that you can commit changes to the jupyter app.
+
+```shell
+git config --global user.email hpcadmin@localhost
+git config --global user.name "HPC Admin"
+```
+
+### Get Jupyter Working
+
+#### Configure the correct cluster
+
+The example application we've created does not use the correct cluster configuration, so we've got
+to modify it.
+
+We need to edit the `form.yml` in the appication's folder. We can navigate to the folder through the
+files app.  The URL is `https://localhost:3443/pun/sys/files/fs/home/hpcadmin/ondemand/dev/jupyter/`.
+
+Here you'll see the `form.yml` file. We can edit it by clicking on the file and pressing the "Edit"
+button.  This will take us to the [file editor app, with this file open](https://localhost:3443/pun/sys/file-editor/edit/home/hpcadmin/ondemand/dev/jupyter/form.yml)
+
+IMG NEEDED
+
+In the file Editor, specify `hpc` as the cluster attribute on line 11 like so: `cluster: "hpc"`. Save this file by clicking
+the "Save" button at the top left.
+
+IMG NEEDED
+
+#### Launch the Jupyter Application
+
+Now when we navigate back to our [interactive sessions](https://localhost:3443/pun/sys/dashboard/batch_connect/sessions),
+you'll see the "Interactive Apps \[Sandbox\]" menu with an item labeled "Jupyter Notebook".
+
+IMG NEEDED
+
+[Follow this link](https://localhost:3443/pun/sys/dashboard/batch_connect/dev/jupyter/session_contexts/new) and we'll be
+presented with this form for specifying different attributes about the job we want to submit to the SLURM scheduler.
+
+IMG NEEEDED
+
+We don't need to change anything in this form, so simply press "Launch" at the bottom of the form. After pressing
+launch the job should have successfully launched the job and redirected us back
+the [interactive sessions](https://localhost:3443/pun/sys/dashboard/batch_connect/sessions) page where we'll
+see a panel showing our job.  
+
+Initially the panel will be grey because our job is being scheduled and jupyter is starting.  When it is
```

Comment:
Insert here the step of the panel switching immediately to complete after starting. Explain this is an indication of an error, show how to access the results and view the output log, seeing that the jupyter command is not found. Go back to the app and edit the template/script.sh to fix the environment by adding:

    source /usr/local/jupyter/2.1.4/bin/activate